drivers/mpu9250/MPU9250.hpp

```diff
@@ -240,6 +243,8 @@ class MPU9250: public ImuSensor
 		_mag_enabled(mag_enabled),
 #if defined(__DF_EDISON)
 		_packets_per_cycle_filtered(4.0f), // The FIFO is supposed to run at 1kHz and we sample at 250Hz.
+#elif defined(__DF_RPI_SINGLE)
+		_packets_per_cycle_filtered(8.0f),// The FIFO is supposed to run at 1kHz and we sample at 250Hz.
```

Comment:
@julianoes  Please tell me how to format the style of the code in DriverFramework?

I'm trying to format the code ,but it says

 ~/src/Firmware/Tools/fix_headers.sh drivers/mpu9250/MPU9250.hpp
/Users/secondstupid/src/Firmware/Tools/fix_headers.sh: ERROR: Your version of awk (4.1.4) is broken. Please use version 4.1.1 or lower.

secondstupid@appledeMacBook-Air-2:DriverFramework$ awk --version
GNU Awk 4.1.4, API: 1.1 (GNU MPFR 3.1.5, GNU MP 6.1.2)
Copyright (C) 1989, 1991-2016 Free Software Foundation.